tests/tools/onert_train/src/args.cc

```diff
@@ -184,6 +198,17 @@ void Args::Initialize(void)
     ("nnpackage", po::value<std::string>()->notifier(process_nnpackage), "NN Package file(directory) name")
     ("modelfile", po::value<std::string>()->notifier(process_modelfile), "NN Model filename")
     ("path", po::value<std::string>()->notifier(process_path), "NN Package or NN Modelfile path")
+    ("data_length", po::value<int>()->default_value(-1)->notifier([&](const auto &v) { _data_length = v; }), "Data length number")
+    ("load_input:raw", po::value<std::string>()->notifier(process_load_raw_inputfile),
+         "NN Model Raw Input data file\n"
+         "The datafile must have data for each input number.\n"
+         "If there are 3 inputs, the data of input0 must exist as much as data_length, "
+         "and the data for input1 and input2 must be held sequentially as data_length.\n"
```

Comment:
From this description, I would prepare input0, input1, ... input{data_length-1}.
But your comment seems to require input_0, input_1, ... input_{data_length-1}.
Also, if the model gets multiple inputs input0_0, input0_1, ...input_{data_length-1}, ... That is, the model input index comes first, and training example index follows with `_`. Let's update the explanation if necessary in next PR.